ui/src/pages/dags.tsx

```
@@ -34,7 +35,21 @@ const Dags: Component = () => {
+                dag={dag}
+                onDelete={() => {
+                  getDagPageCount()
+                    .then((count) => {
+                      setMaxPage(count);
+                      queryClient.invalidateQueries({
+                        queryKey: ["dags"],
+                      });
+                    })
+                    .catch((error) => console.error(error));
+                }}
+              />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)